### ISSUE TITLE
Some EXIF data is dropped from TIFF -> AVIF

### ISSUE THREAD

**Issue by aentwist** [darktable-org/darktable#15038]
### Describe the bug

Using workflow,

darktable -TIFF-> Photoshop -TIFF-> darktable -AVIF->

The 2nd-stage TIFF has EXIF data that darktable drops when exporting to AVIF.

### Steps to reproduce

1. darktable exports TIFF - [IMG_3496.tif](https://github.com/aentwist/darktable-exif-repro/raw/main/IMG_3496.tif)
1. Photoshop exports TIFF - [IMG_3496-2.tif](https://github.com/aentwist/darktable-exif-repro/raw/main/IMG_3496-2.tif)
1. export AVIF from TIFF (IMG_3496-2.tif) in darktable (10-bit, HLG Rec2020) - [IMG_3496-2.avif](https://github.com/aentwist/darktable-exif-repro/raw/main/IMG_3496-2.avif)

---

##### darktable TIFF

`exiftool IMG_3496.tif | findstr Date`

```
File Modification Date/Time     : 2023:08:08 20:19:10-04:00
File Access Date/Time           : 2023:08:08 20:19:27-04:00
File Creation Date/Time         : 2023:08:08 20:18:55-04:00
Modify Date                     : 2023:08:07 17:59:53
Create Date                     : 2022:02:07 07:36:05
Profile Date Time               : 2023:08:04 22:52:23
Date/Time Original              : 2022:02:07 07:36:05
Create Date                     : 2022:02:07 07:36:05.06-04:00
Date/Time Original              : 2022:02:07 07:36:05.060-04:00
Modify Date                     : 2023:08:07 17:59:53.06-04:00
```

##### Photoshop TIFF

`exiftool IMG_3496-2.tif | findstr Date`

```
File Modification Date/Time     : 2023:08:08 20:19:30-04:00
File Access Date/Time           : 2023:08:08 20:21:30-04:00
File Creation Date/Time         : 2023:08:08 20:19:15-04:00
Modify Date                     : 2023:08:07 18:18:22
Metadata Date                   : 2023:08:07 18:18:22-04:00
Date Created                    : 2022:02:07
Date/Time Original              : 2022:02:07 07:36:05
Create Date                     : 2022:02:07 07:36:05
Profile Date Time               : 2023:08:04 22:52:23
Create Date                     : 2022:02:07 07:36:05.06-04:00
Date/Time Original              : 2022:02:07 07:36:05.060-04:00
Modify Date                     : 2023:08:07 18:18:22.06-04:00
Date/Time Created               : 2022:02:07 07:36:05
```

##### darktable AVIF

`exiftool IMG_3496-2.avif | findstr Date`

```
File Modification Date/Time     : 2023:08:08 20:21:10-04:00
File Access Date/Time           : 2023:08:08 20:22:00-04:00
File Creation Date/Time         : 2023:08:08 20:21:10-04:00
Profile Date Time               : 2023:08:04 22:52:23
```

### Expected behavior

darktable should preserve the EXIF data

### Where did you install darktable from?

darktable.org

### darktable version

4.4.0

### What OS are you using?

Windows

### What is the version of your OS?

Windows 10 Education

### Are you using OpenCL GPU in darktable?

None

**Comment by kmilos** [comment]
Have you read the release notes regarding AVIF export?

**Comment by Nilvus** [comment]
So not an issue. I let you read release notes (in Github darktable homepage and check your export preferences. And think about upgrading darktable, We are on 4.4.2 and some bugs had been fixed on 4.4.1 and 4.4.2

**Comment by aentwist** [comment]
> When exporting to AVIF, EXR, JPEG XL, or XCF, selecting specific
metadata (e.g. geo tag or creator) is not currently possible. For
AVIF, EXR, JPEG XL, and XCF formats, darktable will not include any
metadata fields unless the user selects all of the checkboxes in the
export preference options.

Confirmed, thank you
